RULES.md

```diff
@@ -136,26 +136,29 @@ There are no warnings for this rule pack.
-| [iam-user-mfa-enabled](https://docs.aws.amazon.com/config/latest/developerguide/iam-user-mfa-enabled.html)           | AWS Identity and Access Management users have multi-factor authentication (MFA) enabled.                | IA-2(1)(2)(11)        |
-| [iam-password-policy](https://docs.aws.amazon.com/config/latest/developerguide/iam-password-policy.html)             | The account password policy for IAM users meets the specified requirements indicated in the parameters. | AC-2(1)               |
-| [iam-group-has-users-check](https://docs.aws.amazon.com/config/latest/developerguide/iam-group-has-users-check.html) | IAM groups have at least one IAM user.                                                                  | AC-2(j)               |
+| AWS Config Rule Name                                                                                                 | Summary                                                                                                 | Relevant Control IDs                 |
+| -------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- | ------------------------------------ |
+| [ec2-imdsv2-check](https://docs.aws.amazon.com/config/latest/developerguide/ec2-imdsv2-check.html)                   | The EC2 instance has IMDSV2 (instance metadata service version 2) enabled.                              | AC-6                                 |
+| [efs-in-backup-plan](https://docs.aws.amazon.com/config/latest/developerguide/efs-in-backup-plan.html)               | EFS is in an AWS Backup plan.                                                                           | CP-9(b), CP-10, SI-12                |
+| [iam-user-mfa-enabled](https://docs.aws.amazon.com/config/latest/developerguide/iam-user-mfa-enabled.html)           | AWS Identity and Access Management users have multi-factor authentication (MFA) enabled.                | IA-2(1)(2)(11)                       |
+| [iam-password-policy](https://docs.aws.amazon.com/config/latest/developerguide/iam-password-policy.html)             | The account password policy for IAM users meets the specified requirements indicated in the parameters. | AC-2(1)                              |
+| [iam-group-has-users-check](https://docs.aws.amazon.com/config/latest/developerguide/iam-group-has-users-check.html) | IAM groups have at least one IAM user.                                                                  | AC-2(j)                              |
+| [redshift-require-tls-ssl](https://docs.aws.amazon.com/config/latest/developerguide/iam-group-has-users-check.html)  | Redshift clusters require TLS/SSL encryption.                                                           | AC-17(2), SC-7, SC-8, SC-8(1), SC-13 |
```

Comment:
We can check for this rule using the same rule in the [AwsSolutions pack](https://github.com/cdklabs/cdk-nag/blob/main/src/AwsSolutions/rules/databases/RS1.ts)